src/store/modules/user/actions.ts

```diff
@@ -80,15 +80,44 @@ const actions: ActionTree<UserState, RootState> = {
   /**
    * Get User profile
    */
-  async getProfile ( { commit }) {
+  async getProfile ( { commit, dispatch }) {
     const resp = await UserService.getProfile()
     if (resp.status === 200) {
-      const userPref =  await UserService.getUserPreference({
-        'userPrefTypeId': 'SELECTED_BRAND'
-      });
-      const brands = JSON.parse(process.env.VUE_APP_BRANDS)
-      const userPrefBrand = brands.find((brand: any) => brand.id == userPref.data.userPrefValue)
-      commit(types.USER_BRAND_UPDATED, userPrefBrand ? userPrefBrand.id: brands ? brands[0].id : {})
+      const payload = {
+        "inputFields": {
+          "storeName_op": "not-empty"
+        },
+        "fieldList": ["productStoreId", "storeName"],
+        "entityName": "ProductStore",
+        "distinct": "Y",
+        "noConditionFind": "Y"
+      }
+
+      const storeResp = await UserService.getEComStores(payload);
+      if(storeResp.status === 200 && !hasError(storeResp) && storeResp.data.docs?.length > 0) {
+        const stores = storeResp.data.docs;
+
+        resp.data.stores = [
+          ...(stores ? stores : []),
+          {
```

Comment:
This is not needed, we will not support None in preorder app

Reply:
Done sir

Reply:
Done sir